### DISCUSSION TITLE
velocity_embedding_stream cropping dots

### DISCUSSION THREAD

**Discussion by seigfried** [theislab/scvelo#764] [Q&A]
Hello

Thank you for this package. Could you please help? I tried to make a velocity_embedding_stream plot and the dots often get cut at every side.
![image](https://user-images.githubusercontent.com/20457676/144355458-af4ffc4e-bcd4-42ff-b126-1a704027b688.png)

Try 1 : I tried to increase the margins. I tried left_margin, right_margin
```
scv.pl.velocity_embedding_stream(adata3, color_map=mpl.cm.Reds, legend_loc="right",
                                 left_margin=0.2, color="Specific_CellType", basis='umap', fontsize=20, dpi=300, figsize=[8,8])
```
And i get an error:
![image](https://user-images.githubusercontent.com/20457676/144355757-47677528-0551-4062-b071-1e46956a62ec.png)

When i type in`?scv.pl.velocity_embedding_stream` I do see these options.

I realise the figure is also getting cropped from the top so just left, right margins wont work

Try 2 : I tried to simply reduce the dot size. This works but the plot appears faded. Is there a way to not make the dots transparent?

My matplotlib version is 3.4.1 if that helps.
Thank you for your time!

**Comment by WeilerP** [comment]
@seigfried, sorry, never encountered this problem before. Would you please open an issue for this providing all the info asked for, i.e. an example code snippet I can simply copy and paste, the stack trace (in text format, please) and the package versions you are using? I'll then take a look at it ASAP. Thanks!
